src/EditorFeatures/Core/SymbolSearch/SymbolSearchUpdateEngineFactory.cs

```diff
@@ -25,7 +24,11 @@ internal static class SymbolSearchUpdateEngineFactory
                 var client = await workspace.TryGetRemoteHostClientAsync(cancellationToken).ConfigureAwait(false);
                 if (client != null)
                 {
-                    return new RemoteUpdateEngine(workspace, client, logService, cancellationToken);
+                    var session = await client.TryCreateServiceKeepAliveSessionAsync(WellKnownServiceHubServices.RemoteSymbolSearchUpdateEngine, logService, cancellationToken).ConfigureAwait(false);
+                    if (session != null)
+                    {
+                        return new RemoteUpdateEngine(workspace, session);
```

Comment:
FYI, this code has changed a bunch in master.  you are going to conflict if you check it into dev15.6.  can you help figure out why integration tests are failing with OOP in 15.6.  then we can merge master into 15.6 and then you can rebase against that.